pegasus/migrations/106_add_popularityrank_to_tutorials.rb

```diff
@@ -0,0 +1,7 @@
+Sequel.migration do
+  change do
+    alter_table(:tutorials) do
+      add_column :popularityrank, Integer, null: false
```

Comment:
Ah, seems reasonable - I forgot this is one of the tables that's created fresh every time.

Come to think of it, though, aren't all these tables created directly from the Gsheets anyways, so that migrations aren't even necessary? Or am I misremembering how the Gsheets are handled? @ashercodeorg this came up a week or two ago, didn't it?